app/assets/stylesheets/utilities/layout.scss

```diff
@@ -14,6 +14,10 @@
   display: inline-block;
 }
 
-.dashboard-colour {
+.dashboard-background {
```

Comment:
I would pull these into a separate file in `app/assets/stylesheets/components`, maybe `dashboard.scss`. That then needs an extra import statement in [`application.scss`](https://github.com/alphagov/govwifi-admin/blob/master/app/assets/stylesheets/application.scss) and [`application_ie8.scss`](https://github.com/alphagov/govwifi-admin/blob/master/app/assets/stylesheets/application_ie8.scss).